capstone-frontend/src/components/BookSearchList.jsx

```diff
@@ -7,26 +7,51 @@ class BookSearchList extends Component {
   constructor(props) {
     super(props);
     this.state = {
-      bookList: []
+      bookSearchList: []
     }
-    this.getData = this.getData.bind(this);
   }
 
-  getData() {
+  getData = () => {
     fetch(`/api/search?searchTerm=${this.props.searchQuery}`)
       .then(response => response.json())
-      .then(res => this.setState({ bookList: res }))
+      .then(res => this.setState({ bookSearchList: res }))
       .catch(err => console.log(err));
   }
 
+  getUserBookLists = () => {
+    let userBookLists = [{ id: "1", name: "Best Books" },
+    { id: "2", name: "Fantasy" },
+    { id: "3", name: "Sci-fi" }];
+    return userBookLists;
+  }
+
+  getUserClubs = () => {
+    let userClubs = [{ id: "1", name: "Cool Kats Book Club" },
+    { id: "2", name: "Sci-fi Fans" }];
+    return userClubs;
+  }
+
+  getUserCommunities = () => {
+    let userCommunities = [{ id: "1", name: "Sci-fi Community" },
+    { id: "2", name: "Read-a-lot" }];
+    return userCommunities;
+  }
+
   componentDidMount() {
```

Comment:
This structure is great! Will be super easy to swap out your fake data for fetch requests in the future. This is all organized really well.

Reply:
Thank you!